test/session.test.js

```diff
@@ -185,6 +185,117 @@ describe(`session: ${printPath("[test/session.test.js]")}`, function () {
         assert.strictEqual(frontendInfo.up["rsub"], "testuserid");
     });
 
+    // check if access token cookie is cleared if refresh token api is called without the refresh token
```

Comment:
Please add a test to see what happens if there are multiple cookies, but `olderCookieDomain` is undefined

Reply:
done